src/scripts/directives.js

```diff
@@ -319,16 +321,28 @@
                         minuteMin: '0',
                         minuteMax: '60',
                         minuteStep: '1',
-                        useDuration: 'false'
+                        useDuration: false,
+                        padHours: false,
+                        padMinutes: false
                     });
 
                     // Return the (post) link function
                     return function (scope, element, attrs) {
+                        var padZeros = function (number) {
+                            return ('00' + number).slice(-2);
+                        };
+
                         scope.hourChoices = _.range(scope.hourMin, scope.hourMax, scope.hourStep);
+                        if (scope.padHours) {
+                            scope.hourChoices = _.map(scope.hourChoices, padZeros);
+                        }
+
                         scope.minuteChoices = _.range(scope.minuteMin, scope.minuteMax, scope.minuteStep);
-                        var group = ['hours', 'minutes'];
+                        if (scope.padMinutes) {
+                            scope.minuteChoices = _.map(scope.minuteChoices, padZeros);
```

Comment:
That occured to me also, but I think it doesn't matter as when it is processed back, the number conversion will be the same for prefixed zeroes?